data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/cloudwatch_logs/config/CloudWatchLogsSinkConfig.java

```diff
@@ -77,4 +95,81 @@ public int getWorkers() {
         return workers;
     }
 
+    public Map<String, String> getCustomHeaders() {
+        return customHeaders;
+    }
+
+}
+
+/**
+ * Custom validation annotation for HTTP headers.
+ */
+@Documented
+@Constraint(validatedBy = ValidCustomHeaders.CustomHeadersValidator.class)
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+@interface ValidCustomHeaders {
```

Comment:
Java standards call for one class per file. Please move this into another file in the same directory named `ValidCustomHeaders.java`.